app/src/main/cpp/src/appMain/build_config.txt

```diff
@@ -8,7 +8,7 @@ BUILD_BT_SUPPORT:BOOL=OFF
 BUILD_USB_SUPPORT:BOOL=OFF
 
 //Security Ford protocol protection
-ENABLE_SECURITY:BOOL=OFF
+ENABLE_SECURITY:BOOL=ON
```

Comment:
@LitvinenkoIra is this a revert of previous changes? Probably it was always OFF even on 'develop'

Reply:
Yes, this is a revert of https://github.com/CustomSDL/sdl_core/commit/5b8fd9f2b6aedfc45d8affbe92a6343517060356#diff-5a93f5d02a4621cbd0107108dd84cd459884638998dd051f431d9ad59ddaf699